e2e_test/streaming/bug_fixes/stack_overflow_17342.slt

```diff
@@ -1,5 +1,3 @@
-# Why this is release mode only: https://github.com/risingwavelabs/risingwave/issues/21355#issuecomment-2806876248
```

Comment:
Can we keep link to issue? So if we encounter some regression again, we have a starting point to debug.